lib/url_parser.rb

```diff
@@ -30,12 +30,13 @@ def self.parse_to_full_user_url(url)
   def self.try_all(url)
     GithubURLParser.parse_to_full_url(url) ||
     GitlabURLParser.parse_to_full_url(url) ||
-    BitbucketURLParser.parse_to_full_url(url)
+    BitbucketURLParser.parse_to_full_url(url) ||
+    ApacheSvnUrlParser.parse_to_full_url(url)
   end
 
   def parse_to_full_url
     path = parse
-    return nil unless path.present?
+    return nil if path.nil?
```

Comment:
Maybe also check for `|| path.empty?` since `path.present?` would be `false` in that case

Reply:
Updated in 4565f0f512c5967d20ab20cf98c89501ed648b54